slider.css

```diff
@@ -0,0 +1,57 @@
+.slider {
+  display: block;
+}
+
+.slider-container {
+  display: inline-block;
+  position: relative;
+}
+
+.image {
```

Comment:
not sure if style image should belong to slider. It looks like more general class that can be reused without a slider